modules/scripts/genindex.py

```diff
@@ -80,22 +77,16 @@ def get_index(root, subprojects, sub_docpaths, javadoc):
     for (dirpath, dirnames, filenames) in os.walk(os.curdir):
         _ignore_files(dirpath, filenames, ['README.rst', 'README.md'], root)
         _ignore_dirs(dirpath, dirnames, ['yaydoctemp', 'yaydocclone', '.git'])
-        _ignore_dirs(dirpath, dirnames, ['source'] + subproject_dirs, os.curdir)
```

Comment:
Don't remove this entirely. Replace with `_ignore_dirs(dirpath, dirnames, ['source'], os.curdir)`